src/category_theory/category/Cat.lean

```diff
@@ -49,6 +50,13 @@ def objects : Cat.{v u} ⥤ Type u :=
 { obj := bundled.α,
   map := λ C D F, F.obj }
 
+/-- Any isomorphism in `Cat` induces an equivalence of the underlying categories. -/
+def equiv_of_iso {C D : Cat} (γ : C ≅ D) : C.α ≌ D.α :=
+{ functor := γ.hom,
+  inverse := γ.inv,
+  unit_iso := eq_to_iso $ eq.symm γ.hom_inv_id',
+  counit_iso := eq_to_iso γ.inv_hom_id' }
```

Comment:
I'm still worried this isn't the right definition, but failing to articulate my objection exactly. I worry that it might be better to "push the proofs down as far as possible". That is, use `nat_iso.of_components` to construct `unit_iso` and `counit.iso`, and specify that the components are given by `eq_to_iso` applied to `γ.hom_inv_id` "restricted" to a particular object.